cluster-up/.gitignore

```diff
@@ -0,0 +1 @@
+cluster/kind-k8s-sriov*/csrcreator/*.cert
```

Comment:
@rmohr maybe this too?

Reply:
this file is coming from kubevirt/kubevirtci. We have to keep it.